src/core/Settings.js

```diff
@@ -1361,7 +1361,16 @@ function Settings() {
                 mode: Constants.CMCD_MODE_QUERY,
                 enabledKeys: Constants.CMCD_AVAILABLE_KEYS,
                 includeInRequests: ['segment', 'mpd'],
-                version: 1
+                version: 1,
+                targets: [
+                    {
+                        mode: 'response',
+                        enabled: false,
+                        url: 'http://example.com/report1',
+                        enabledKeys: ['tr', 'd', 's'],
+                        includeOnRequests: ['mpd', 'segment']
+                    }
+                ]
```

Comment:
```suggestion
                version: 1,
                targets: []
```

The target's default should be an empty array.

 Also, you should add to the JSDocs the targets structure like:
 ```
 * @property {Array.<target>} [targets]
 *  The targets....
 *
  * @property {module:Settings~CmcdSettings} target
 *  The target....
 *
 ```
 
 This is at the top of the Settings file.